contrib/start_minitest

```diff
@@ -5,7 +5,4 @@ echo "Preparing application..."
 bundle exec rake dev:bootstrap[old_test_suite] RAILS_ENV=test
 echo "Running api test suite..."
 export NO_MEMCACHED=1
-bundle exec rake test:api
-tmp_exit=$?
-rm -rf .bundle vendor/bundle
-exit $tmp_exit
+bundle exec ruby test/functional/attributes_test.rb # --name=test_create_namespace_old -v
```

Comment:
oops - having an env variable to control this would be super cool. Having to patch local git sucks